src/targets/betafpvf411rx/target.h

```diff
@@ -23,6 +23,30 @@
 #define BUZZER_PIN PIN_A14
 //#define BUZZER_INVERT
 
+//--------------------------------------------------------------
+// RGB LEDs  --- Not currently working --
+//--------------------------------------------------------------
+#define RGB_LED_NUMBER 0
+//--------------------------------------------------------------
+// DMA => TIM3-CC4 => DMA1, Channel5, Stream2 (See drv_dma.c)
+//--------------------------------------------------------------
+#define RGB_LED_DMA        DMA_DEVICE_TIM3_CH4
+//--------------------------------------------------------------
+// pin / port for the RGB led
+//--------------------------------------------------------------
+#define RGB_PIN LL_GPIO_PIN_1
+#define RGB_PORT GPIOB
```

Comment:
please use the `PIN_B1` method of specifying these, that will also allow the drv_gpio helpers to work.